frameworks/digital-outcomes-and-specialists/questions/briefs/essentialRequirementsOutcomes.yml

```diff
@@ -1,17 +1,18 @@
-name: Essential skills and experience
+name: 'Technical competence: essential skills and experience'
```

Comment:
Yeah it will. It's not a permanent solution. We're going do more work on this problem but Sarah wants to do something short term to emphasise the relationship between everything under technical competence.